sub/osd.c

```diff
@@ -79,9 +84,10 @@ const struct m_sub_options osd_style_conf = {
         .font = "sans-serif",
         .font_size = 55,
         .color = {255, 255, 255, 255},
-        .border_color = {0, 0, 0, 255},
-        .shadow_color = {240, 240, 240, 128},
-        .border_size = 3,
+        .outline_color = {0, 0, 0, 255},
+        .back_color = {240, 240, 240, 128},
```

Comment:
Since `osd-back-color` and `osd-shadow-color` are the same now, this causes the back color of the osd bar to not be fully transparent anymore.